src/plugins/irc-events/banlist.js

```diff
@@ -9,7 +9,7 @@ module.exports = function(irc, network) {
 	irc.on("banlist", function(banlist) {
 		const channel = banlist.channel;
 		const bans = banlist.bans;
-		if (!bans) {
+		if (!bans || bans.length === 0) {
```

Comment:
It appears that we now get through an empty array rather than undefined when there are no bans. So this needs updated. This was added in [irc-framwork](https://github.com/kiwiirc/irc-framework/commit/1cc8892f6bf5414e677f0d37daa5984b77b58374).